lib/compat/wordpress-6.0/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -17,6 +17,52 @@
  */
 class WP_Theme_JSON_Resolver_Gutenberg extends WP_Theme_JSON_Resolver_5_9 {
 
+	/**
+	 * Given a theme.json structure modifies it in place
+	 * to update certain values by its translated strings
+	 * according to the language set by the user.
+	 *
+	 * @param array  $theme_json The theme.json to translate.
+	 * @param string $domain     Optional. Text domain. Unique identifier for retrieving translated strings.
+	 *                           Default 'default'.
+	 * @return array Returns the modified $theme_json_structure.
+	 */
+	protected static function translate( $theme_json, $domain = 'default' ) {
```

Comment:
This method is not modified with respect to `WP_Theme_JSON_Resolver_5_9::translate`, though it needs to live here to read the `theme-i18n.json` from the `lib/compat/6.0` folder.